pyscript.core/esm/custom-types.js

```diff
@@ -124,6 +124,8 @@ export const handleCustomType = (node) => {
                             afterRunAsync: codeAfterRunWorkerAsync,
                         });
 
+                        module.setGlobal(interpreter, "XWorker", XWorker);
```

Comment:
Just so I understand/learn this means that xWorker has its own global?

Reply:
Answering, just so @WebReflection can tell me I'm an idiot and to improve my own understanding.

AFAICT, this simply means that `XWorker` is an object within the Python `globals` object. No need to `from js import XWorker` because it's already there. Right..? The `setGlobal` just takes the `interpreter` and a `"Name"` and binds it in `global` scope to the JS `Object` that's the third argument. Right?

Reply:
That makes sense 😄  Thank you for explaining @ntoll

Reply:
Leaving for context: MicroPython *DOES* have a `registerJSModule`.

Reply:
> Moreover, they are implemented in Python

here they are implemented by the module, so "grab them from js" is not even an option.

Wasmoon doesn't have `registerJSModule` and so doesn't Ruby, I won't be blocked by this aesthetic discussion until the TP, but hey, **PR welcome**.